test/invoice_repository_test.rb

```diff
@@ -101,9 +101,42 @@ def test_finds_all_by_updated_at
     assert_equal 0, result2.length
   end
 
+  def test_create_creates_a_new_invoice
+    load_data
+    new_invoice = @se.invoice_repository.create(
+      customer: @se.customer_repository.customers[0],
+      merchant: @se.merchant_repository.merchants[0],
+      status: "shipped",
+      items: [@se.item_repository.items[0],
+              @se.item_repository.items[0],
+              @se.item_repository.items[1]]
+    )
+    assert_equal 1, new_invoice.customer_id
+    assert_equal 100, new_invoice.merchant_id
+  end
+
   private
 
   def fixtures_directory
     File.expand_path('../../data/fixtures', __FILE__)
   end
+
+  def load_data
+    repo_data = {
+        items: [
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.